components/coloring-listgroup.vue

```diff
@@ -29,22 +33,24 @@
 
 <script>
 
-	import { BListGroup, BListGroupItem } from "bootstrap-vue";
-
 	export default {
 
-		components: {
-
-			"b-list-group": BListGroup,
-			"b-list-group-item": BListGroupItem
-		},
-
 		data() {
 
 			return {
 
-				clickedOpacity: "1.0",
-				defaultOpacity: "0.5"
+				listGroupItemClass: "coloring-listgroup-item",
+
+				// Preset opacity values for swapping opacities on click
+				// Default opacities are set via initial class
+				opacities: {
+
+					clicked: "1.0",
+					default: "0.5"
+				},
+
+				// Index of most recently clicked list group item
+				clickedIndex: 0
```

Comment:
Sure, this small change is possible though currently moot since `clickedIndex` is only used as check to see if the same category is being clicked again. More on that in your next comment on `null` and `currentCategory` below.